assets/scss/_custom.scss

```diff
@@ -82,6 +82,23 @@ body.td-home main[role="main"] > section:first-of-type .content p:first-child {
   margin-bottom: 1.5em;
 }
 
+// Style search on main (landing) page
+body.td-home section.k8s-search {
+  margin-left: 0;
+  margin-right: 0;
+  padding: 3rem;
+  padding-left: calc(max(2rem, (100vw - 44rem) / 2));
+  padding-right: calc(max(2rem, (100vw - 44rem) / 2));
```

Comment:
Making the search input on https://k8s.io/ take up less than the full viewport width.